_vector-search/optimizing-storage/binary-quantization.md

```diff
@@ -117,6 +117,53 @@ PUT my-vector-index
 ```
 {% include copy-curl.html %}
 
+## Enhancing search quality with ADC and RR
+
+If the recall is lacking, you can specify asymmetric distance computation (ADC) or random rotation (RR) in the index mapping to apply enhanced search capabilities. These techniques are available in OpenSearch 3.2 and later versions.
```

Comment:
```suggestion
If your search results have low recall, you can apply enhanced search capabilities by specifying asymmetric distance computation (ADC) or random rotation (RR) in the index mapping.
```

Reply:
How about "If your search results have low recall, you can enhance search quality by specifying asymmetric distance computation (ADC) or random rotation (RR) in the index mapping."?